test/rss.cpp

```diff
@@ -711,4 +711,21 @@ TEST_CASE("If item's <title> is empty, try to deduce it from the URL",
 	REQUIRE(feed->items()[4]->title() == "Alternate link isn't first");
 }
 
+TEST_CASE("rss_feed::is_query_feed()", "[rss]")
```

Comment:
Please provide a description here. If you wanted me to write that method and you could only use one sentence to explain it to me, what would it be? :)

Reply:
I would probably leave that to you as a thing to find out... and write a proper description. ;)

Sure, I will change them.